pkl-commons-cli/src/main/kotlin/org/pkl/commons/cli/commands/BaseOptions.kt

```diff
@@ -142,28 +130,11 @@ class BaseOptions : OptionGroup() {
     option(
         names = arrayOf("--ca-certificates"),
         metavar = "<path>",
-        help = "Replaces the built-in CA certificates with the provided certificate file."
+        help = "Trust CA certificates from the provided file."
```

Comment:
I chose this wording because this option replaces both ~/.pkl/cacerts and built-in certificates, which seemed too long to explain here. How about "Only trust CA certificates from the provided file(s)."? (The option is repeatable.)

Reply:
New wording LGTM